docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
 
```

Comment:
This commit shouldn't be necessary. The changelog page is a landing page of sorts of the the OSS app releases, which is still at `13.3.3`.